frontend/src/metabase/collections/components/CollectionLink.jsx

```diff
@@ -1,9 +1,14 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
```

Comment:
I'd encourage to move this file to frontend/src/metabase/collections/components/CollectionLink/CollectionLink.styled.jsx

as it's all about styled components.

I'd also simplify what is now lines 28-40, maybe in another PR.

Reply:
We can also start using the `space` function for all those paddings of 8px and so on.

Reply:
What do you think if we do that in a different PR? Mixing big refactorings and file changes like that could make bisecting more difficult

Reply:
Yes, sure.